packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -1,31 +1,17 @@
 import type Koa from 'koa';
-import { errors } from '@strapi/utils';
-import {
-  validateEntryContentType,
-  validateReleaseActionCreateSchema,
-  validateUniqueEntryInRelease,
-} from './validation/release-action';
-import { ReleaseActionCreateArgs, UserInfo } from '../types';
+import { validateReleaseActionCreateSchema } from './validation/release-action';
+import { ReleaseActionCreateArgs } from '../../../shared/types';
+import { getService } from '../utils';
 
 const releaseActionController = {
   async create(ctx: Koa.Context) {
-    const user: UserInfo = ctx.state.user;
     const releaseActionArgs: ReleaseActionCreateArgs = ctx.request.body;
 
-    // Check permissions: releases can only be managed by super admins for now
-    if (!strapi.admin.services.role.hasSuperAdminRole(user)) {
-      throw new errors.ApplicationError('Content Releases is a superadmin only feature');
-    }
+    await validateReleaseActionCreateSchema(releaseActionArgs);
 
-    await Promise.all([
-      validateReleaseActionCreateSchema(releaseActionArgs),
-      validateEntryContentType(releaseActionArgs),
-      validateUniqueEntryInRelease(releaseActionArgs),
-    ]);
-
-    const releaseActionService = strapi.plugin('content-releases').service('release-action');
-    // TODO: releaseAction is of type any when the create service return is of type Promise<GetValues<"plugin::content-releases.release-action", string>> ...?
-    const releaseAction = await releaseActionService.create(releaseActionArgs);
+    const releaseService = getService('release', { strapi });
```

Comment:
you're using the global strapi var here right? Is there a way to avoid that?

Reply:
Yeah I guess that's true so I could also just not pass it...